storage/innobase/xtrabackup/src/fil_cur.cc

```diff
@@ -102,6 +106,41 @@ const char *xb_get_relative_path(
   }
 }
 
+/** check for a tablespace in data dictonary
+@in tablespace_name database/tablespace name to check
+@in space_id 	space_id to check
+return true if found or false
+*/
+static bool check_for_tablespace_in_dd(std::string tablespace_name,
```

Comment:
Also DD related checks, we should put in dict0dd.cc
dict_tablespace_exists_dd()? or
dd_tablespace_exists()?

Another convention I would like us to follow is to prefix  functions introduced by us in other source files with 'xb_'
(cc: @marce ^)

So maybe xb_dd_tablespace_exists().